src/components/sbb-icon/sbb-icon.tsx

```diff
@@ -108,18 +106,27 @@ export class SbbIcon implements ComponentInterface {
     }
   }
 
-  public render(): JSX.Element {
-    return (
-      <Host role="img" data-namespace={this._svgNamespace} data-empty={!this._svgIcon}>
-        {this._svgIcon ? (
-          <span class="sbb-icon-inner" innerHTML={this._svgIcon}></span>
-        ) : (
-          <span class="sbb-icon-inner">
-            {/* To reserve space, we need an empty svg to apply dimension to. */}
+  protected override render(): TemplateResult {
+    // ## Host attributes ##
```

Comment:
Nitpick; I'm not sure the comment is generally needed?

Reply:
I don't really have a strong opinion on that, I'm fine if we remove it after the migration script has run. 👍